tempodb/backend/json.go

```diff
@@ -0,0 +1,25 @@
+package backend
+
+import "sync"
+
+var (
+	dedicatedColumnsKeeper = map[string]*DedicatedColumns{}
```

Comment:
agree there's nothing better than a global var b/c of the way Unmarshal/Marshal works, but i'd like this data to have a lifecycle.

Should we add a "ClearDedicatedColumnsCache()" (or whatever) method that wipes this map called right before polling?

Reply:
Good call.  I put this in the defer of the poller.Do so that we clear the memory when we're done with polling.  I can see putting it at the beginning instead, but then we'd hold the memory even after we're done polling.  Could do both I suppose, but this is probably okay.